sync_auditlog/models/log.py

```diff
@@ -0,0 +1,313 @@
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import uuid
+import psycopg2
+import io
+import ast
+
+from odoo import fields, models
+from datetime import datetime
+try:
+    from xmlrpc import client as xmlrpclib
+except ImportError:
+    import xmlrpclib
+
+def _generate_xml_id(self, skip=False):
```

Comment:
@vimalpatelserpentcs 
This does the same as `__ensure_xml_id()`, so I see no point in having it, and can remote.